docs/data-ai/data/visualize.md

```diff
@@ -35,42 +35,152 @@ See [configure a machine](/operate/get-started/supported-hardware/) for more inf
 
 {{% /expand%}}
 
-### Configure a workplace with a sensor widget
+### Create a workspace
 
-{{< table >}}
-{{% tablestep number=1 %}}
-**Create a workspace on Viam**
+1. Navigate to the **FLEET** page's [**TELEOP** tab](https://app.viam.com/teleop).
+   Click **+ Create workspace**.
+   Enter a unique name for your workspace in the top left of the page, replacing the placeholder `example-workplace` text.
 
-Log in to [Viam](https://app.viam.com/).
+   {{<imgproc src="/services/data/visualize-choose-workspace.png" resize="800x" style="width: 700px" class="fill imgzoom shadow" declaredimensions=true alt="On the TELEOP page, you can choose an existing workspace or create a new workspace.">}}
 
-Navigate to the **FLEET** page's **TELEOP** tab.
-Create a workspace by clicking **+ Create workspace**.
-Give it a name.
+1. From the **Select location** dropdown, select the location that contains the machine that you would like to visualize data from.
 
-{{<imgproc src="/how-tos/teleop/blank-workspace.png" resize="800x" style="width: 500px" class="fill imgzoom shadow" declaredimensions=true alt="Blank teleop page.">}}
+   {{<imgproc src="/services/data/visualize-choose-location.png" resize="800x" style="width: 700px" class="fill imgzoom shadow" declaredimensions=true alt="Choosing a location for your workspace.">}}
 
-{{% /tablestep %}}
-{{% tablestep number=2 %}}
-**Add widgets**
+1. From the **Select machine** dropdown, select the machine that you would like to visualize data from.
 
-Click **Add widget** and add a **GPS** widget for any position-reporting sensor, a **stat** widget for current readings from a sensor, or a **time series** widget to graph data for any component that supports capturing numeric data.
-Use the widget header to configure the panel.
-Repeat as many times as necessary.
+   {{<imgproc src="/services/data/visualize-choose-machine.png" resize="800x" style="width: 700px" class="fill imgzoom shadow" declaredimensions=true alt="Choosing a machine for your workspace.">}}
 
-{{% /tablestep %}}
-{{% tablestep number=3 %}}
-**Select a machine**
+### Add a widget
 
-Now, select a machine with which to make your teleop workspace come to life.
-Click **Select machine** and select your configured machine.
+1. Click **Add widget** and select a widget type to create a new widget of that type on your workspace:
 
-Your dashboard now shows the configured widget for the data from your machine.
-For example, a time series graph measuring noise over time for a sensor component:
+   {{<imgproc src="/services/data/visualize-add-widget.png" resize="800x" style="width: 300px" class="fill imgzoom shadow" declaredimensions=true alt="Choosing a widget type from the widget menu.">}}
```

Comment:
the first 3 screenshots in this section also seem unnecessary since the UI is pretty clear

Reply:
I disagree completely. The widget configuration and movement buttons have no text, no tooltip, and confused me on first use. Removed the first screenshot since that UI is clear enough.

Reply:
Hmm maybe where it says "Use the widget header to configure" suggest to app that they make it "Use the edit button to configure" or generally change the UI?